docs/src/administration/users.md

```diff
@@ -3,136 +3,184 @@ title: "User administration"
 
-* **Viewer** - Viewer can view all environments of this type.
-* **Contributor** - Contributor can push code and branch all environments of this type.
-* **Administrator** - Administrator can change settings and execute actions on all environments of this type.
 
-{{< note theme="warning" title="Important" >}}
-After you add or remove a user from a project or an environment type, you will need to trigger a redeploy to propagate the access changes to each environment.
-You can redeploy by using the CLI command `platform redeploy` or the `Redeploy` button in the management console. 
-{{< /note >}}
+## Environment types
 
-If you want your contributors to be able to see everything, but only commit to environments of a certain type, set their permission as **Project Viewer** for the whole project and update their permission on that environment type to **Contributor**.
+Each environment type groups one or more environments together so that you can manage access for all environments of a certain type.
+This allows you to set permissions for multiple environments at once based on their purpose.
 
-{{< note >}}
-The project owner - the person licensed to use Platform.sh - has the project administrator role. The project owner is the only user that can make changes to the project plan. 
-{{< /note >}}
+Platform.sh offers three environment types: Production, Staging, and Development.
+You can assign user permissions for each environment type.
+Any permissions you assign to an environment type apply to all environments of that type.
 
-## SSH access
+For example, if you assign User1 **Admin** permissions for Development environments,
+User1 has **Admin** permissions for all environments of that type.
 
-By default, everyone with access equal or greater than **Contributor** can access the project through SSH. 
+A few things to consider:
 
-To customize who can use SSH, [set the `access` key](/configuration/app/access.md) in your `.platform.app.yaml` file.
+* Only one environment per project can be the Production type. It's set automatically as the default branch and can't be overridden separately.
+* You can change an environment's type (if it's not Production).
+* You can have multiple Staging and Development environments.
 
-## Manage user permissions in the console
+The following table shows the available roles for environment types.
 
-From your list of projects, select the project where you want to view or edit user permissions. Click the Settings tab at the top of the page, then click the `Access` tab on the left to see the users in the project and their roles.
+| Role | View environment | Push code | Branch environment | SSH access | Change settings | Execute actions |
+| ---- | ---------------- | --------- | ------------------ | ---------- | --------------- | --------------- |
+| Environment Type Viewer
```

Comment:
I thought the idea was to standardize names with what's in the UI. the "Environment Type" part isn't, so it seems somewhat confusing. I think it's an overly long name for the role.
Also the ones below